src/app/newcomers/newcomers.component.html

```diff
@@ -0,0 +1,123 @@
+      </p>
+    </div>
+    <div class="row text-center">
+      <img class="container-fluid mb-5" src="/assets/img/full_map.png">
+    </div>
+    <div>
+      <p class="text-center">
+        At Systers, we welcome contributions of all forms. Non-coders are welcomed to join the community!
+        <br> Apart from code you can contribute in any of the following ways:
+      </p>
+    </div>
+    <div class="container box logos mx-auto d-block mb-5">
+      <div class="row text-center justify-content-center">
+        <div class="card col-sm-3 mt-3 mb-3 ml-3 mr-3 pt-2" *ngFor="let det of method_details">
+          <img class="card-img-top" src="{{det.image}}" alt="{{det.methodName}}">
+          <div class="card-body">
+            <h5 class="card-title">{{det.methodName}}</h5>
+            <button type="button" class="btn btn-primary waves-light mx-auto" (click)="style.show()" mdbWavesEffect>View Details</button>
+            <!-- Modal -->
+            <div mdbModal #style="mdb-modal" class="modal fade" id="centralModalSuccess" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
+              aria-hidden="true">
+              <div class="modal-dialog modal-notify modal-success" role="document">
+                <div class="modal-content">
+                  <div class="modal-header" style="background-color: #8d2165;">
+                    <p class="heading lead" style="background-color: #8d2165 ">{{det.methodName}}</p>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close" (click)="style.hide()">
+                      <span aria-hidden="true" class="white-text">×</span>
+                    </button>
+                  </div>
+                  <div class="modal-body">
+                    <img class="card-img-top mt-3 mb-5 h-50" src="{{det.image}}" alt="{{det.methodName}}">
+                    <div class="text-center">
+                      <p>{{det.details}}</p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <!-- Modal -->
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container box logos mx-auto d-block mt-5 mb-5">
+      <div class="row text-center justify-content-center">
+        <h2 class="mt-4 mb-4">
+          How to get started with contributing to Open Source?
+        </h2>
+      </div>
+      <div class="row text-center justify-content-center">
+        <h2>OSS Workflow Diagram</h2>
+      </div>
+      <div *ngFor="let stage of workflow_stages">
+        <div class="row text-center justify-content-center mt-5">
+          <button type="button" class="btn btn-link mx-auto" (click)="style.show()" mdbWavesEffect>
+            <h3>{{stage.name}}</h3>
+          </button>
+          <!-- Modal -->
+          <div mdbModal #style="mdb-modal" class="modal fade" id="centralModalSuccess" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
```

Comment:
more appropriate name than style?

Reply:
Can you explain this more? :)

Reply:
I assume style is the name you are using to refer to the modal on line 73 in which case a more appropriate name than style would be something that is not generic and sounds like a keyword.